src/esp/bindings/SceneBindings.cpp

```diff
@@ -40,7 +40,8 @@ void initSceneBindings(py::module& m) {
       .value("EMPTY", SceneNodeType::EMPTY)
       .value("SENSOR", SceneNodeType::SENSOR)
       .value("AGENT", SceneNodeType::AGENT)
-      .value("CAMERA", SceneNodeType::CAMERA);
+      .value("CAMERA", SceneNodeType::CAMERA)
+      .value("OBJECT", SceneNodeType::OBJECT);
```

Comment:
Perhaps make these Pascal case instead of all caps?

Reply:
Should probably be refactored on master and then migrated.